### DISCUSSION TITLE
DonationSummary: filter 'give_payment_gateway_donation_summary' applied twice

### DISCUSSION THREAD

**Discussion by Genevieve-K** [impress-org/givewp#6673] [Developer Q&A] [answered]
In src/Framework/PaymentGateways/DonationSummary.php, the getSummaryWithDonor() function applies a filter on the summary. But before the function getSummary() was called with the same filter. I add an information in this description, and this information is added twice.

**Comment by JasonTheAdams** [comment, marked as answer]
Resolved in #6678
